x-pack/test/fleet_api_integration/config.event_ingested.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrConfigProviderContext } from '@kbn/test';
+
+export default async function ({ readConfigFile }: FtrConfigProviderContext) {
+  const baseFleetApiConfig = await readConfigFile(require.resolve('./config.base.ts'));
+  const serverArgs: string[] = [
+    ...baseFleetApiConfig.get('kbnTestServer.serverArgs'),
+    // serverless oblt needs only event.ingested, without agent id verification
+    `--xpack.fleet.agentIdVerificationEnabled=false`,
+    `--xpack.fleet.eventIngestedEnabled=true`,
+  ];
```

Comment:
If this configuration is for serverless, I would like to bring few thoughts:

- to access `samlAuth` service this config and loaded tests should be moved into `x-pack/test_serverless` directory
- using `supertest` service in tests for API calls is strongly not recommended as it is authenticated with `operator` user, not available for our customers on MKI; SAML auth with required role is the proper way to simulate end-user experience.

More details: https://github.com/elastic/kibana/tree/main/x-pack/test_serverless#roles-based-testing
Feel free to raise questions in #appex-qa